providers/trello.js

```diff
@@ -0,0 +1,700 @@
+// trello.js
+// https://developers.trello.com/apis/webhooks
+// ========
+module.exports = {
+    parse: function (req, discordPayload) {
+        //In order to actually see the json payload.
+        console.log(req.body);
+        console.log(req.body.action.data);
```

Comment:
Just a friendly reminder to delete these console.log before it get merged ;)
For further testing purposes you can use the website [RequestBin](https://requestb.in/) to receive the payloads